simulator/src/sequential/DflipFlop.js

```
@@ -42,6 +42,9 @@ export default class DflipFlop extends CircuitElement {
      * WIP always resolvable?
      */
     isResolvable() {
+        if(this.reset.value == 1 && this.preset.value == 1){
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)